api/controllers/tasks.py

```diff
@@ -120,6 +125,23 @@ def process_proposal(credentials, tpid):
             last_updated=db.sql.func.now(),
         )  # Annotation config is sentiment example.
 
+        emailReceiver = tp_creator_email
```

Comment:
small nitpick but don't see the point of making this new `emailReceiver` variable when you can just do `contacts=[tp_creator_email]` on line 136 below, and also I think as far as possible we're using `snake_case`